TF_Settings_Web/src/TouchFree/Connection/MessageReceiver.ts

```diff
@@ -250,20 +230,16 @@ export class MessageReceiver {
     ClearUnresponsivePromises(): void {
         let lastClearTime: number = Date.now();
 
-        if (this.responseCallbacks !== undefined) {
-            for (let key in this.responseCallbacks) {
-                if (this.responseCallbacks[key].timestamp < lastClearTime) {
-                    delete this.responseCallbacks[key];
-                } else {
-                    break;
-                }
-            };
-        }
+        MessageReceiver.ClearUnresponsiveItems(lastClearTime, this.responseCallbacks);
```

Comment:
How do we feel about typescript inferring vs explicit types for more complex generics?
e.g.
`MessageReceiver.ClearUnresponsiveItems<WebSocketResponse>(lastClearTime, this.responseCallbacks);`
vs
`MessageReceiver.ClearUnresponsiveItems(lastClearTime, this.responseCallbacks);`